### ISSUE TITLE
[EXAMPLE] Hacker News Reader

### ISSUE THREAD

**Issue by ghalestrilo** [revery-ui/revery#603]
Hi! I noticed some missing examples on your [roadmap](https://github.com/revery-ui/revery/wiki/Roadmap), and I'm interested in taking on the Hacker News Reader.
I searched for issues but none seem related to that. Can I make a fork?

**Comment by lessp** [comment]
By coincidence I just brought @bryphe's example back to life. Was planning on getting it done by the weekend. Maybe there could be a combined effort! 🤷‍♂ 

![image](https://user-images.githubusercontent.com/17602389/66873399-a39c8980-efa8-11e9-805f-6ed45ac858cb.png)

**Comment by ghalestrilo** [comment]
No way, that's too cool! Is seems pretty close to done, though. Let me know if I can actually help with anything in this project! Otherwise, I'll try to find another way to contribute :)

**Comment by lessp** [comment]
I've continued and pushed the work here. 

It's not finished yet, but has the basic functionality let me know where you'd like this when it's done @bryphe!

As said previously, this is just continued work on top of the previous efforts here: <https://github.com/revery-ui/revery-fetch/tree/example/initial-hn-app/example>

**Comment by OhadRau** [comment]
Would be really cool to get this merged into the examples so that we can track regressions & keep it up-to-date with new Revery APIs. Thoughts on putting this in `examples/` and hooking it up to one of the tabs?

**Comment by lessp** [comment]
> Would be really cool to get this merged into the examples so that we can track regressions & keep it up-to-date with new Revery APIs. Thoughts on putting this in `examples/` and hooking it up to one of the tabs?

Yes, definitely, I've been holding off to move it to the Revery-organisation since it's still moving quite a bit. I've been thinking of using GraphQL instead of basic client-requests, and also add HTTPS-support for Windows which is why I've held off!

**Comment by lessp** [comment]
Picking this up again, I'm thinking of moving the started example to the Revery organisation, despite previous comments. It'd be helpful to have people contribute as well.

**Comment by lessp** [comment]
Moved it: https://github.com/revery-ui/revery-hacker-news
